core/src/main/thrift/client.thrift

```diff
@@ -46,13 +46,13 @@ enum TableOperationExceptionType {
   NOTFOUND
   OFFLINE
   BULK_BAD_INPUT_DIRECTORY
-  BULK_BAD_ERROR_DIRECTORY
```

Comment:
Can you explain this rename to me?  Not sure what is going on here in the larger context.

Reply:
I got the idea from https://diwakergupta.github.io/thrift-missing-guide/#_versioning_compatibility, see the 3rd bullet.

Reply:
Should the same be done for the enum in manager.thrift?

Reply:
I initially thought that was not needed, as it may have been something used internally only. But, it looks like I might need to do that.

Reply:
I took a quick look i the thrift code and it seems like it may use intergers for the enums on the wire, so the obsolete things makes sense for that reason.  If it used the enum names on the wire it would not matter so much.